### PR TITLE
fix - AccuracyCallback for multi-label classification

### PR DESCRIPTION
**Pull request by lightforever**
criterion/accuracy now can work only with a multi-class classification problem.

Use case: multi-label classification problem

targets: batch_size x num_classes, each sample has binary vector where more than one 1 is possible

outputs: batch_size x num_classes, logits

criterion: BCEWithLogitsLoss

To cope with that, we can check targets.size(1), if it's > 1 that is a multi-class classification problem.

topk can not be applied in that case.

### CODE REVIEW

**Comment by Scitator** [comment]
Could you please describe the usecase for such feature?
And why dont use just sever AccuracyCallbacks?
like [here](https://github.com/catalyst-team/classification/blob/master/configs/exp_splits_augs.yml#L118)

or... Are you talking about multi-label classification problem?

**Comment by lightforever** [comment]
Pardon, yes I am talking about multi-label classification problem.

Use case: 

targets: batch_size x num_classes, each sample has binary vector where more than one 1 is possible

outputs:  batch_size x num_classes, logits

criterion: BCEWithLogitsLoss

Callback configuration:
```
    accuracy:
      callback: AccuracyCallback
      threshold: 0.5
      activation: Sigmoid
```

**Comment by Scitator** [comment]
Hmm, now I see, looks reasonable.
Nevertheless, as @TezRomacH says, could you please write mini docs for `accuracy`, because now it looks a bit complicated. For example, there is no way to use `topk` in you case and it can be confusing.

**Comment by lightforever** [comment]
@Scitator , I have added docs to Accuracy callback and criterion/accuracy.

How do you think, is it enough?
